Makefile

```diff
@@ -114,10 +114,14 @@ KUSTOMIZE=$(shell pwd)/bin/kustomize
 kustomize: ## Download kustomize locally if necessary.
 	$(call go-get-tool,$(KUSTOMIZE),sigs.k8s.io/kustomize/kustomize/v4@v4.5.5)
 
-YQ = $(shell pwd)/bin/yq
+YQ=$(shell pwd)/bin/yq
 yq: ## Download yq locally if necessary.
 	$(call go-get-tool,$(YQ),github.com/mikefarah/yq/v4@latest)
 
+E2E=$(shell pwd)/bin/e2e
+skywalking-e2e: ## Download e2e locally if necessary.
+	$(call go-get-tool,$(E2E),github.com/apache/skywalking-infra-e2e/cmd/e2e@2a33478)
```

Comment:
does the cmd tool support installation with a version tag rather than a commit hash?

Reply:
according to the latest [docs](https://skywalking.apache.org/docs/skywalking-infra-e2e/latest/en/setup/install/#install-via-go-install) the commit hash is preferred. But I just found out that it has already been released v1.2.0 and we can use the v1.2.0 tag here.